OSFeels first draft of CFP language.

roseaboveit opened the pull request.

roseaboveit committed:
OSFeels first draft of CFP language.

jnf commented on db/texts/event_info.md:
```diff
@@ -1,37 +1,53 @@
+### About You
 
-The proposal form is divided into four sections, three of which are used as criteria for acceptance. 
+You have feelings. You care about open source. Maybe you have thoughts on how we can work better in our open source communities. Maybe you have thoughts on how our humanity impacts our OS work, or vice versa. Perhaps you've read through our wishlist and something caught your eye. When you heard about this conference you had an idea that you just knew would fit.
```
Ideally, this would link to our wishlist. :)

jnf commented on db/texts/event_info.md:
```diff
@@ -1,37 +1,53 @@
+
+### Topics
+
+We are interested in practical information that our participants can take back with them to their Open Source communities. We are interested in authentic presentations opening up needed conversations in the larger community. Our wishlist may be helpful to get the ideas flowing.
```
Again, linky when available.

jnf commented on db/texts/event_info.md:
```diff
@@ -1,37 +1,53 @@
+
+### Proposal Details
+
+Pick a format for your presentation, such as a formal talk, panel, activity, spoken word, etc.
```
Can we add musical or dramatic performance to this list? I'm nudging toward what I'd love to see. :)

jnf commented on db/texts/event_info.md:
```diff
@@ -1,37 +1,53 @@
+
+Pick a format for your presentation, such as a formal talk, panel, activity, spoken word, etc.
+
+Choose a length for your presentation between 10 and 40 minutes.
```
I don't think 10 and 40 were hard limits. Maybe update to something like, "Choose a length for your presentation. Most are between 10 and 40 minutes, but we will consider other options."

jnf commented on db/texts/event_info.md:
```diff
@@ -1,37 +1,53 @@
+
+Choose a length for your presentation between 10 and 40 minutes.
+
+You can submit multiple talks, but speakers will only speak once at the event.
```
I'm hoping not every presenter can be collected as "speaker." Can we use a more generic term like "..., but each presenter will only present once at the event," or something similar?

jnf commented on db/texts/event_info.md:
```diff
@@ -1,37 +1,53 @@
+
+### Proposal Form
+
+The proposal form is divided into four sections, three of which are used as criteria for acceptance.
```
We dropped the `Help Us Out` section, so now there's only three. :)

jnf commented on db/texts/event_info.md:
```diff
@@ -1,37 +1,53 @@
+
+The proposal form is divided into four sections, three of which are used as criteria for acceptance.
+
+_Talk Information_ is the primary data used during proposal evaluation and selection. Please help us to remain as unbiased as possible by not including personally identifying information in this section such as pronouns, websites, or company names.
```
"unbiased as possible" feels awkward to me. Maybe "Please help us avoid unconscious/unintentional bias by..?"

jnf commented:
Added some notes, but overall this is awesome. :)

jnf committed:
The commit incorporates my comments on #1.

This tweaks some language to (hopefully) be more clear and does some minor formatting cleanup.

jnf commented:
Sorry for the force push _and_ for the commit which blatantly just adds my notes. If we're gonna stay on schedule, we need to merge and deploy this. We can tweak it later via the admin interface; this is definitely good enough to get us going.
